modules/sdk-coin-eth/src/lib/utils.ts

```diff
@@ -579,7 +579,7 @@ type RecursiveBufferOrString = string | Buffer | RecursiveBufferOrString[];
 export function getRawDecoded(types: string[], serializedArgs: Buffer): RecursiveBufferOrString[] {
   function normalize(v: unknown, i: number): unknown {
     if (BN.isBN(v)) {
-      return v.toBuffer();
+      return v;
     } else if (typeof v === 'string' || Buffer.isBuffer(v)) {
       return v;
     } else if (Array.isArray(v)) {
```

Comment:
are we confident that it will not fail on the following ifs?

Reply:
No other problems were reported, so it seems like Avax is the only coin that somehow triggers decoding of something to `BN`.